src/pages/policies/BackupPolicy.tsx

```diff
@@ -247,7 +260,6 @@ export const BackupPolicy = ({ type }) => {
                   component={Select}
                   optionLabel="description"
                   dataKey="name"
-                  required
```

Comment:
required is broken, it's impossible to submit form with expanded tab which contains required field
this field and the one in BackupDestinationComponent.tsx has default value anyway set by react state